src/pages/guides/using-local-validator.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Using the Local Validator
+metaTitle: Using the Local Validator
+description: Learn how to setup a local development environment and use a local validator
+# remember to update dates also in /components/guides/index.js
+created: '08-9-2024'
+updated: '08-9-2024'
+---
+
+## Overview
+
+A **Local Validator** acts as your personal node, providing a local sandbox environment for testing applications without the need to connect to a live blockchain network. It operates a **fully customizable local test ledger**, which is a simplified version of the Solana ledger, equipped with all **native programs pre-installed** and various features enabled.
+
+### Setup 
+
+To start using the local validator, you'll need to install the Solana Tools CLI using the appropriate commands for your operating system.
+
+{% dialect-switcher title="Installation Commands" %}
+
+{% dialect title="MacOs & Linux" id="MacOs & Linux" %}
+
+```
+sh -c "$(curl -sSfL https://release.solana.com/v1.18.18/install)"
+```
+
+{% /dialect %}
+
+{% dialect title="Windows" id="Windows" %}
+
+```
+cmd /c "curl https://release.solana.com/v1.18.18/solana-install-init-x86_64-pc-windows-msvc.exe --output C:\solana-install-tmp\solana-install-init.exe --create-dirs"
+```
+
+{% /dialect %}
```

Comment:
this might be quite frustrating since later we say that it only works for windows when using WSL. Maybe we should recommend WSL here already insted of the .exe?

Reply:
I made the same comment in a private conversation.

Reply:
So for this actually you can do everything up to when we create our own bash script to create your own validator with just the CLI (like dumping and loading programs) that's why I still left it because you can still actually run your local validator, will just not be that easy as calling `metaplex-local-validator`